src/main/java/com/smartbear/readyapi/plugin/git/ui/ImportProjectFromGitGui.java

```diff
@@ -25,8 +27,14 @@ public Component getComponent() {
     @Override
     public VcsRepositoryInfo downloadProjectFiles(File emptyDirectory) {
         try {
-            gitCommandHelper.cloneRepository(getSelected().getRepositoryPath(), getSelected().getCredentialsProvider(), emptyDirectory);
-            return new VcsRepositoryInfo("Git", getSelected().getRepositoryPath());
+            if (isLocal()) {
+                new InternalProImportWsdlProjectAction().perform(ApplicationEnvironment.getWorkspace(), emptyDirectory);
```

Comment:
I think it is not a good idea to use any internal ReadyAPI's actions from plugins. Plugin should know nothing about these actions.

Reply:
Also it is strange that something that should download project files may opens project instead of download.

Reply:
Fixed. If the Local option is selected, method does not do and returns nothing.